Makefile

```diff
@@ -33,20 +33,22 @@ demo-swarm:
 	make cluster
 	make install-test-swarm
 
-# Create a release and test the release installation on a vagrant cluster
+# Create a build and test the release installation on a vagrant cluster
+# This should only be used with devbuild, else install-test-kubeadm will fail to
```

Comment:
Isn't it expected to be used with non devbuilds for the case where we want to test the release before uploading it?